src/new/wallet.js

```diff
@@ -533,7 +715,96 @@ class HathorWallet extends EventEmitter {
     };
   }
 
+  /**
+   * Get balance for a token (same as old method to be used for compatibility)
+   *
+   * @params {string} tokenUid Token uid
+   *
+   * @return {Object} Object with balance { available, locked }
+   *
+   * @memberof HathorWallet
+   * @inner
+   **/
+  getOldBalance(tokenUid) {
+    storage.setStore(this.store);
+    const uid = tokenUid || this.token.uid;
+    const historyTransactions = this.getOldHistory();
+    return wallet.calculateBalance(Object.values(historyTransactions), uid);
+  }
+
+  /**
+   * Get full wallet history (same as old method to be used for compatibility)
+   *
+   * @return {Object} Object with transaction data { tx_id: { full_transaction_data }}
+   *
+   * @memberof HathorWallet
+   * @inner
+   **/
+  getOldHistory() {
```

Comment:
Same here. What is changing that we need a compatibility method?

Reply:
This method is still used as an auxiliar method in the whole facade and it's needed because the return is different from the wallet service return, it's much simpler. I agree that `getOldHistory` is a bad name, should I call it `getSimpleHistory`?

Reply:
Maybe `_getRawHistory()`?

Reply:
Changed to `_getHistoryRaw`